source/openqasm/tests/test_printer.py

```diff
@@ -1,8 +1,7 @@
 import dataclasses
 
-import pytest
-
 import openqasm3
+import pytest
```

Comment:
It looks like `pylint` is complaining about this change of import order. You will need to revert to the old one before this gets merged. You can do it by hand (easy enough in this case) or with something like `isort`.

Reply:
For your convenience, https://github.com/Swordcat/openqasm/pull/1/files

Reply:
Sorry about that, should be fixed now